.infrastructure/hpa.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: todoapp
+  namespcae: mateapp
```

Comment:
There is a typo in the 'namespace' field. It should be 'namespace' instead of 'namespcae'.